django/contrib/postgres/indexes.py

```diff
@@ -22,12 +22,12 @@ def deconstruct(self):
         kwargs['pages_per_range'] = self.pages_per_range
         return path, args, kwargs
 
-    def get_sql_create_template_values(self, model, schema_editor, using):
-        parameters = super().get_sql_create_template_values(model, schema_editor, using=' USING brin')
+    def create_sql(self, model, schema_editor, using=''):
+        statement = super().create_sql(model, schema_editor, using=' USING brin')
         if self.pages_per_range is not None:
-            parameters['extra'] = ' WITH (pages_per_range={})'.format(
-                schema_editor.quote_value(self.pages_per_range)) + parameters['extra']
-        return parameters
+            statement.parts['extra'] = ' WITH (pages_per_range={})'.format(
```

Comment:
A bit unrelated, but I would move the closing parenthesis to improve readability:
```
statement.parts['extra'] = ' WITH (pages_per_range={})'.format(
    schema_editor.quote_value(self.pages_per_range)
) + statement.parts['extra']
```